lib/src/resultset-table/ResultsetTable.stories.tsx

```diff
@@ -206,6 +206,25 @@ export const Chromatic = () => (
         <DxcResultsetTable columns={columnsSortable} rows={longValues} />
       </SmallContainer>
     </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Reduced sortable table" theme="light" level={4} />
+      <DxcResultsetTable columns={columnsSortable} rows={rowsSortable} mode="reduced" />
+    </ExampleContainer>
+    {/* PENDING SMALL ICON VERSION */}
```

Comment:
This depends on the implementation of the new table icon Compound Component which is currently being worked in.